django/chat/forms.py

```diff
@@ -169,7 +169,7 @@ class ChatOptionsForm(ModelForm):
     class Meta:
         model = ChatOptions
         fields = "__all__"
-        exclude = ["chat", "global_default"]
+        exclude = ["chat", "global_default", "prompt"]
```

Comment:
I added "prompt" field to ChatOptions model, but don't save it except when saving a preset.

The prompt gets submitted along with the Preset form, not with the ChatOptions form.